src/pages/home/home.jsx

```diff
@@ -7,6 +7,7 @@ import ImageShopping from "../../images/undraw_add_to_cart_vkjp.svg";
 // import ImageToDo from "../../images/undraw_following_q0cr.svg";
```

Comment:
is always better to remove comments if you don't have words explaining why you have those lines commented 🚀